cypress/integration/home_spec.js

```diff
@@ -0,0 +1,25 @@
+describe('The home page', function() {
+    // 'PubMiner' title is on /list
+    it('contains the PubMiner title', function () {
+        cy.visit('/list')
+        cy.title().should('include', 'PubMiner')
+    })
+
+    it('contains have an empty input box', function() {
+        cy.get('input').should('not.have.value', 'US')
+    })
+
+    // TODO: figure out how the date and count will be implemented and check for those things
+    it('displays a last updated date and article count', function(){
+        cy.get('#whatsnew').should('contain', 'Last update')
+        cy.get('#update-count').should('contain','new articles added') //any number
```

Comment:
We should probably display a fixed number of articles in the "What's new section". I'd think the most recent 10 or 20 would be good. From my point of view, the what's new mostly serves to give the user an idea of the currency of the data in our database.